ingestion/src/metadata/ingestion/bulksink/metadata_usage.py

```diff
@@ -179,77 +141,130 @@ def write_records(self) -> None:
                 self.status.failures.append(
                     "Table: {}".format(value_dict["table_entity"].name.__root__)
                 )
+
+    def write_records(self) -> None:
+        for usage_record in self.file_handler.readlines():
+            record = json.loads(usage_record)
+            table_usage = TableUsageCount(**json.loads(record))
+            self.service_name = table_usage.serviceName
+            if "." in table_usage.table:
+                databaseSchema, table = table_usage.table.split(".")[-2:]
+                table_usage.table = table
+                if not table_usage.databaseSchema:
+                    table_usage.databaseSchema = databaseSchema
+            table_entities = self.__get_table_entity(
+                table_usage.database, table_usage.databaseSchema, table_usage.table
+            )
+            for table_entity in table_entities or []:
```

Comment:
add a flag to `@fqn_build_registry.add(Table)` to specify if we want to get back a single FQN or all available FQNs matching the ES search